### DISCUSSION TITLE
How to make 'tab' choose a completion when it is the only one

### DISCUSSION THREAD

**Discussion by ittaysw** [Aloxaf/fzf-tab#396] [Q&A]
Without fzf-tab, If I have a prefix with several completions and hit tab, they are shown. If I then add a few characters and hit tab, and there is now only one completion, it is completed.
With fzf-tab when there are several completions, they are shown. If I then add some search term so that the list is narrowed to one and hit tab, nothing happens. I need to hit enter. 
I would like to have tab enter the selection (when there's a single one or many). Right now it is bound to 'down'

**Comment by TaiSHiNet** [comment]
You can bind tab to accept:
```
zstyle ':fzf-tab:*' fzf-bindings 'tab:accept'
```
But it'll always happen (not only if you have a single result)

**Reply by ColtonIdle** [reply]
noob here. where does this line go? in my zshrc file? or is there a separate place for fzf-tab configuration?

**Comment by Aloxaf** [comment]
Related: https://github.com/Aloxaf/fzf-tab/issues/202

This depends on when `fzf` supports dynamic binding. Currently, you cannot change the binding at runtime.
